src/60_AMP_Ads/_Hello_World.html

```diff
@@ -28,8 +28,7 @@
   <script async src="https://cdn.ampproject.org/amp4ads-v0.js"></script>
   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
   <!--
-    The AMP Ad boilerplate.
-  -->
+    The AMP Ad boilerplate which is a lot shorter than the regular AMP boilerplate. -->
```

Comment:
which is a lot shorter -> which is shorter
if you say "a lot", not sure how much is "a lot"